### ISSUE TITLE
Add support for Hydra database

### ISSUE THREAD

**Issue by ChenpiDog** [dbeaver/dbeaver#21295]
Refer to：https://github.com/hydradatabase/hydra

Use postgre type connection to check the table structure. You cannot distinguish whether it is a row or columnar table.

**Comment by LonwoLonwo** [comment]
Hello @ChenpiDog 

Can the PostgreSQL JDBC driver be used for connection? 
Can you connect to Hydra via the PostgreSQL connector in DBeaver?

**Comment by ChenpiDog** [comment]
@LonwoLonwo Yes, but some details are not fully displayed. For example, is Hydra's table a row table or a column table? Customization would be better.

**Comment by ChenpiDog** [comment]
The "access method" is viewable in psql with \d+ which is just a standard sql query underneath the hood... exposing the access method further would be a DBeaver feature request.

**Comment by LonwoLonwo** [comment]
Ok. Does Hydra have an official JDBC connector?

**Comment by ChenpiDog** [comment]
Hydra would use the Postgres JDBC connector.

**Comment by LonwoLonwo** [comment]
Ok, thanks for the extra info.
